### DISCUSSION TITLE
PCB and case for MiniDexed by mzuelch

### DISCUSSION THREAD

**Discussion by probonopd** [probonopd/MiniDexed#273] [Show and tell]
@mzuelch is working on a PCB and case for MiniDexed in https://github.com/mzuelch/CATs-Desktopsynth:

![image](https://user-images.githubusercontent.com/2480569/172060099-743347fe-693f-4032-ade5-2ff00767e820.png)

I am wondering what the 90 dregrees rotation of the display compared to the Raspberry Pi connector is for.
Doesn't this unnecessarily increase the required space?

**Comment by mzuelch** [comment]
Hi there!
It may increase the volume by a certain amount - but I wanted to have USB on the front and MIDI, Power and sound on the back.
I'm working in a quite tight studio environment and didn't like to cope with jacks sticking out left and right. By rotating the display, I can still use clumsyMIDI for MIDI and sound as well as USB for my keyboard.

**Reply by probonopd** [reply]
Hi @mzuelch thanks for commenting here. Which Raspberry Pi models is this compatible with?

**Reply by mzuelch** [reply]
I've only tried 3B+ so far - my last RPi1 was bricked some time ago and i was never so lucky to find an affordable RPi4

**Reply by probonopd** [reply]
By the way, we are still looking for an official MiniDexed PCB layout... what do you think about our requirements? https://github.com/probonopd/MiniDexed/issues/88

**Reply by probonopd** [reply]
RPi Zero 2 WH seems to be the price/performance sweet spot (under 20 EUR), but it's pretty much unobtainable nowadays

**Reply by mzuelch** [reply]
A generic RPi header for MiniDeXed seems to be straightforward electronically - it's the mechanicals that stopped me from following this path.
The MIDI jacks (mandatory for my setup) are quite high and I would have had to raise the LCD and ENC to an additional board nevertheless. Any LCD module and ENC fixed directly on a dedicated board would lead to a quite interesting but hard to manufacture case. Using clumsy on the other hand was quite straightforward - I simply had to cut out the space requirements for the MIDI jacks and LCD and ENC were in the perfect position.

**Reply by probonopd** [reply]
I am still trying to wrap my head around it - do you happen to have a photo of the final project (without the case)?

**Reply by mzuelch** [reply]
Since I've had to disassemble my build for the upcoming paint job, I cannot send any photos of the actual build atm.
I've just uploaded a JPG of the interior and a 3D PDF of the whole assembly to my github additionally to the STEP files already included:

https://github.com/mzuelch/CATs-Desktopsynth/blob/main/Projects/Minidexed/Mechanics/MiniDeXed%20Case%20Interior.JPG
https://github.com/mzuelch/CATs-Desktopsynth/blob/main/Projects/Minidexed/Mechanics/MiniDeXed%20Case.pdf

I use 2.54mm sockets with elongated pins on the clumsy to route the GPIO to the upper level. Its a big sandwich after all.

![grafik](https://user-images.githubusercontent.com/76059155/172062555-70998c16-e148-4e2e-8717-5aea7a47f0b5.png)

edit: the Audio and DC are wired to an external jack. I've added landing pads for DC plus stripping points in the most recent design. Audio can be taken directly by jack or by the dedicated pins on the PCM board - you dont have to use a header to connect these pins to the clumsy - just leave em as it is and use the holes for your wires.

**Reply by probonopd** [reply]
Ah, now I understand. Quite clever @mzuelch - at least if you already have a clumsyMIDI board.

**Reply by mzuelch** [reply]
There are certain things that ... accumulate ... when you are into DIY synth stuff :)
Keep up the nice work!

**Reply by craigyjp** [reply]
If you want it for a studio environment then you could easily rack mount it. A €15 rack mount case is ideal for doing this.

![IMG_06062022_091250_(1350_x_1080_pixel).jpg](https://user-images.githubusercontent.com/51026305/172105673-3689c2c5-5b3e-49c2-833a-4be8f3e69cf2.jpg)

![IMG_06062022_091219_(1350_x_1080_pixel).jpg](https://user-images.githubusercontent.com/51026305/172105686-f1b19bd4-b434-46ce-a016-d4feedc68426.jpg)

**Comment by diyelectromusic** [comment]
That looks great!  I recognise well that cut-out :)  I did something similar for a DIY MT32-Pi Clumsy MIDI board I built: https://diyelectromusic.wordpress.com/2021/05/11/mt32-clumsy-midi-simple-midi-filter/

I might have to see if I can order one too once you've settled on a design :)

Aside, I did manage to get the Clumsy MIDI SSD1306 display working too as a test, but that relies on a pending update to the circle environment itself.  But of course there was no space left for an encoder... so what I really need is a Clumsy MIDI with an encoder too :)

Kevin

**Reply by mzuelch** [reply]
The design is quite settled and working. But I've never came across a DIY synth project that is actually finished :)
I've just generated the DXF (case) and the Gerber (addon) files to be found in the "production data" subfolder.
The lasercut DXF is for 3mm thick MDF material (via formulor.de).
